From c2207aa2f1984e5050be9ed62e8b45b27b7ed7bd Mon Sep 17 00:00:00 2001
From: Xavier Hanin <xavier@apache.org>
Date: Sun, 1 Apr 2007 17:39:19 +0000
Subject: [PATCH] FIX: minor problem with IVY-458 fix, setFile should only be
 called if we are not doing an inline post resolve

git-svn-id: https://svn.apache.org/repos/asf/incubator/ivy/core/trunk@524642 13f79535-47bb-0310-9956-ffa450edef68
---
 src/java/org/apache/ivy/ant/IvyPostResolveTask.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/java/org/apache/ivy/ant/IvyPostResolveTask.java b/src/java/org/apache/ivy/ant/IvyPostResolveTask.java
index 977fd0ff9..dda9a12e9 100644
--- a/src/java/org/apache/ivy/ant/IvyPostResolveTask.java
+++ b/src/java/org/apache/ivy/ant/IvyPostResolveTask.java
@@ -166,6 +166,7 @@ protected void ensureResolved(boolean haltOnFailure, boolean useOrigin, boolean
         
         if (confs.length > 0)  {
         	IvyResolve resolve = createResolve(haltOnFailure, useOrigin);
+    		resolve.setFile(_file);
         	resolve.setCache(cache);
         	resolve.setTransitive(transitive);
         	resolve.setConf(StringUtils.join(confs, ", "));
@@ -229,7 +230,6 @@ protected IvyResolve createResolve(boolean haltOnFailure, boolean useOrigin) {
 		resolve.setHaltonfailure(haltOnFailure);
 		resolve.setUseOrigin(useOrigin);
 		resolve.setValidate(isValidate());
-		resolve.setFile(_file);
 		return resolve;
 	}
 
